modern_scraper.py

```
@@ -0,0 +1,1414 @@
+            
+            # Close any open time filter dialogs
+            close_buttons = self.driver.find_elements(By.CSS_SELECTOR, 
+                'button[aria-label*="Close"], button[aria-label*="Done"], button[aria-label*="Apply"]')
+            
+            for button in close_buttons:
+                try:
+                    if button.is_displayed():
+                        button.click()
+                        time.sleep(1)
+                        break
+                except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)